devops/docker/debian10-arm64/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM multiarch/qemu-user-static:x86_64-aarch64 as qemu
+FROM arm64v8/debian:buster
```

Comment:
use secure mirror `mcr.microsoft.com/mirror/docker/library/debian:buster`

Reply:
We should onboard the arm64 image - see https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/containers-bburns/microsoft-container-registry/microsoft-artifact-registry-also-known-as-microsoft-container-registry-docs/onboardingdocs/newimageonboarding